### DISCUSSION TITLE
Better GraphQL uploads support

### DISCUSSION THREAD

**Discussion by SergioSuarezGil** [Kong/insomnia#4110] [Ideas] [answered]
Would be nice something useful options like Altair GraphQL uploads
https://www.xkoji.dev/blog/working-with-file-uploads-using-altair-graphql/

![Captura de pantalla 2021-10-14 a las 11 03 16](https://user-images.githubusercontent.com/3636232/137286430-b24257ff-2054-4287-b81d-ffb587edfdf8.png)

**Comment by wdawson** [comment, marked as answer]
Looks like this is a duplicate of #3714 so I'm going to lock this conversation. Feel free to move the conversation there and upvote if you still want to see this feature.
